en-US/firefox/features/index.lang

```diff
@@ -146,10 +157,12 @@ Block ad trackers
 More powerful Private Browsing
 
 
+# Incognito and InPrivate are features offered by Chrome and IE respectively. Feature names should remain in English unless they are localized in your language.
```

Comment:
Second part: "Check a localized version of each software to see how these features are translated, otherwise keep them in English".